src/ripple/ripple.js

```diff
@@ -118,7 +119,12 @@ MaterialRipple.prototype.upHandler_ = function(event) {
 
   // Don't fire for the artificial "mouseup" generated by a double-click.
   if (event && event.detail !== 2) {
-    this.rippleElement_.classList.remove(this.CssClasses_.IS_VISIBLE);
+    // Allow a repaint to occur before removing this class, so the animation
+    // shows for tap events, which seem to trigger a mouseup too soon after
+    // mousedown.
+    window.setTimeout(function() {
```

Comment:
I originally used `setTimeout(fn, 0)`, but I noticed `TINY_TIMEOUT` used a fair bit in other components (possibly for similar issues), so wanted to keep things consistent.